gobblin-hive-registration/src/main/java/org/apache/gobblin/hive/HiveMetastoreClientPool.java

```diff
@@ -59,6 +60,17 @@
 
   public static final String POOL_CACHE_TTL_MINUTES_KEY = "hive.metaStorePoolCache.ttl";
 
+  public static final String POOL_EVICTION_POLICY_CLASS_NAME = "org.apache.commons.pool2.impl.DefaultEvictionPolicy";
```

Comment:
What is the behavior of this eviction policy? Can we add a javadoc to help user understand why we are using this eviction policy? Also, should this be configurable?

Reply:
Addressed